src/modules/transaction/components/TransactionDetailsView/index.js

```diff
@@ -1,53 +1,154 @@
-import React from 'react';
+import React, { useMemo, useState } from 'react';
+import { withRouter } from 'react-router';
+import ReactJson from 'react-json-view';
+import { useTranslation } from 'react-i18next';
 import { isEmpty } from 'src/utils/helpers';
+import { parseSearchParams } from 'src/utils/searchParams';
 import Box from 'src/theme/box';
 import BoxContent from 'src/theme/box/content';
+import Heading from 'src/modules/common/components/amountField/Heading';
 import BoxHeader from 'src/theme/box/header';
-import TransactionDetailsContext from '@transaction/context/transactionDetailsContext';
-import { LayoutSchema } from '../TransactionDetails/layoutSchema';
-import layoutSchemaStyles from '../TransactionDetails/layoutSchema.css';
+import Table from 'src/theme/table';
+import TokenAmount from 'src/modules/token/fungible/components/tokenAmount';
+import DateTimeFromTimestamp from 'src/modules/common/components/timestamp';
 import NotFound from './notFound';
 import styles from './styles.css';
+import TransactionEvents from '../TransactionEvents';
+import { useTransactions } from '../../hooks/queries';
+import TransactionDetailRow from '../TransactionDetailRow';
+import header from './headerMap';
 
-const TransactionDetails = ({
-  t, activeToken, network, title,
-  transaction: { error, isLoading, data }, wallet,
-  containerStyle,
-}) => {
-  if (!error && isEmpty(data)) {
-    return <div />;
-  }
+const TransactionDetails = ({ location }) => {
+  const transactionID = parseSearchParams(location.search).transactionId;
+  const { t } = useTranslation();
+  const [isParamsCollapsed, setIsParamsCollapsed] = useState(false);
+
+  const {
+    data: transactions,
+    error,
+    isLoading,
+    isFetching,
+  } = useTransactions({
+    config: {
+      params: {
+        transactionID,
+      },
+    },
+  });
+
+  const transaction = useMemo(() => transactions?.data?.[0] || {}, [transactions]);
+  const transactionDetailList = useMemo(() => {
+    if (error || isEmpty(transactions?.data)) return [];
 
-  if (error && isEmpty(data)) {
+    const {
+      id,
+      moduleCommandName,
+      sender = {},
+      nonce,
+      fee,
+      block = {},
+      confirmations,
+      executionStatus,
+    } = transaction;
+    const [txModule, txType] = moduleCommandName.split(':');
+
+    return [
+      {
+        label: t('Transaction type'),
+        value: `${txModule} ${txType}`,
+        isCapitalized: true,
+      },
+      {
+        label: t('Sender'),
+        value: sender,
+        type: 'address',
+      },
+      {
+        label: t('Transaction Fee'),
+        value: <TokenAmount val={fee} token="LSK" />,
```

Comment:
Can we make the token name dynamic?

Reply:
so I have added to todo to this. since we don't have this parameter as part of the response payload for now @soroushm suggested we leave it like this and add a todo to it